addons/website/static/src/snippets/s_website_form/options.js

```diff
@@ -34,7 +66,22 @@ const FormEditor = options.Class.extend({
             return field.records;
         }
         // Set selection as records to avoid added conplexity
-        if (field.type === 'selection') {
+        if (field._property && field.type === "tags") {
+            field.type = "many2many";
+            field.records = field.tags.map(tag => ({
+                id: tag[0],
+                display_name: tag[1],
+            }));
```

Comment:
That was my initial thought as well, but the stored value for `tags` properties is their lower-case identifier.
If you look inside the database, it looks like this:
```sql
# select task_properties from project_task where task_properties is not null;
             task_properties              
------------------------------------------
 {"bf63cbad941a1c36": ["first", "third"]}
(1 row)
```
This is also the reason we need a specific parsing for `tags` (otherwise we could have used the one that existed for `one2many` which expected integers)
<https://github.com/odoo/odoo/pull/119834/files#diff-dfcfd76018e37e31f68f340b8a3413a903382ca35df536fdaae127310e2b50feR119-R120>